internal/command/command.go

```diff
@@ -26,7 +26,7 @@ type Command struct {
 type templateInputs struct {
 	LocalPort int
 	Args      *Args
-	Outputs   map[string]Output
+	Outputs   map[string]string
```

Comment:
Create `type Outputs map[string]string` to hold output collection behavior. Add a `Stdout` method or similar to it that returns a `map[string]string`.